src/SFA.DAS.ApprenticeFeedback.Api/Controllers/ApprenticeFeedbackResultController.cs

```diff
@@ -56,11 +56,15 @@ public async Task<IActionResult> GetApprenticeFeedbackDetails(long ukprn)
         }
 
         [HttpGet("reviews")]
-        public async Task<IActionResult> GetApprenticeFeedbackRatingSummary()
+        public async Task<IActionResult> GetApprenticeFeedbackRatingSummary([RegularExpression("^(|All|AY\\d{4})$", ErrorMessage = "Time period should be empty, 'All', or in the format 'AYdddd'")] string timePeriod)
```

Comment:
I see that there is no validation in this code base, so you are introducing it for the first time, we typically use FluentValidation - have you chosen not to do that for a reason?